components/Footer.js

```diff
@@ -132,56 +198,66 @@ class Footer extends React.Component {
-          </Container>
-          <Container
-            display="flex"
-            justifyContent="space-evenly"
-            alignItems="center"
-            width={1}
-            my={3}
-            order={['2', null, '3']}
-            maxWidth="300px"
+            <Container
+              display="flex"
+              justifyContent="space-evenly"
+              alignItems="center"
+              width={1}
+              my={4}
+              order={['2', '3', null, null, '2']}
+              maxWidth="288px"
+            >
+              <SocialLink href="https://blog.opencollective.com/">
+                <Blog size={17} color="#76777A" />
+              </SocialLink>
+              <SocialLink href="https://twitter.com/opencollect">
+                <Twitter size={17} color="#76777A" />
+              </SocialLink>
+              <SocialLink href="https://github.com/opencollective">
+                <Github size={17} color="#76777A" />
+              </SocialLink>
+              <SocialLink href="https://slack.opencollective.com">
+                <Slack size={17} color="#76777A" />
+              </SocialLink>
+              <SocialLink href="mailto:info@opencollective.com">
+                <Mail size={19} color="#76777A" />
+              </SocialLink>
+            </Container>
+          </Flex>
+          <Flex
+            width={[1, null, null, null, '804px']}
+            as="nav"
+            flexWrap="wrap"
+            justifyContent="center"
+            mt={3}
+            mx={[null, 3]}
+            flex={['1 1 auto', null, null, null, 'none']}
           >
-            <SocialLink href="https://blog.opencollective.com/">
-              <Blog size={17} color="#9399A3" />
-            </SocialLink>
-            <SocialLink href="https://twitter.com/opencollect">
-              <Twitter size={17} color="#9399A3" />
-            </SocialLink>
-            <SocialLink href="https://github.com/opencollective">
-              <Github size={17} color="#9399A3" />
-            </SocialLink>
-            <SocialLink href="https://slack.opencollective.com">
-              <Slack size={17} color="#9399A3" />
-            </SocialLink>
-            <SocialLink href="mailto:info@opencollective.com">
-              <Mail size={19} color="#9399A3" />
-            </SocialLink>
-          </Container>
-          <Flex as="nav" flexWrap="wrap" justifyContent="center" mt={3} flex="1 1 auto" order={['3', null, '2']}>
             {Object.keys(navigation).map(key => (
-              <Box key={key} width={[0.5, null, 0.25]} mb={3}>
-                <P textAlign={['center', null, 'left']} fontSize="1.2rem" color="black.800" letterSpacing="1px" pb={3}>
+              <Box key={key} width={[0.5, 0.25]} mb={3}>
+                <P
+                  textAlign={['center', 'left']}
+                  fontSize="10px"
+                  fontWeight="600"
+                  lineHeight="15px"
+                  color={['black.800', 'black.500']}
```

Comment:
```suggestion
                  color="black.800"
```

Reply:
I think there's a color change in the screen sizes based on the design

Reply:
Design was updated, `black.500` doesn't meet the contrast standards